src/main/java/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.FileReaderService;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> readFile(String fileName) {
+        List<String> dataInput = new ArrayList<>();
```

Comment:
there is ready to use method Files.readAllLines(), internally it uses BufferedReaer, so we can use it here